tests/snapshots/element__it_can_parse_deeply_nested.snap

```diff
@@ -62,7 +62,7 @@ expression: dom
                                   ],
                                   "children": [
                                     "this is deep",
-                                    "hello world"
+                                    "hello world\n                            "
```

Comment:
There are some cases where we get this ugly output. But this seems to be the same thing browsers do when you try to edit text nodes where they also seem to include line breaks and stuff.

Altogether I think this is the better behavior as you can just trim the result if you don't need the whitespaces.